extensions/PhabBugz/Extension.pm

```diff
@@ -54,6 +58,34 @@ sub webservice {
   $args->{dispatch}->{PhabBugz} = "Bugzilla::Extension::PhabBugz::WebService";
 }
 
+sub object_end_of_update {
+  my ($self,   $args)    = @_;
+  my ($object, $changes) = @$args{qw(object changes)};
```

Comment:
To make sure I understand this right: `$object` is a BMO user model, right?
Can we name it `$user` instead?

Or, alternatively, call it `$object`, but after the `!$object->isa('Bugzilla::User')` guard, then do `my $user = $object`? Might make this easier to follow 🤔